python/samples/04.ai.f.dataSource.azureOpenAI/README.md

```diff
@@ -0,0 +1,85 @@
+# Azure OpenAI On Your Data
+
+The following is a conversational bot that uses the Azure OpenAI Chat Completions API `Azure OpenAI on Your Data` feature to facilitate RAG (retrieval augmentation) using Azure AI Search as the Azure data source.
+
+<!-- @import "[TOC]" {cmd="toc" depthFrom=1 depthTo=6 orderedList=false} -->
+
+<!-- code_chunk_output -->
+
+-   [Azure OpenAI On Your Data](#azure-openai-on-your-data)
+    -   [Summary](#summary)
+        -   [Example Interaction](#example-interaction)
+    -   [Setting up the sample](#setting-up-the-sample)
+    -   [Testing the sample](#testing-the-sample)
+        -   [Using Teams Toolkit for Visual Studio Code](#using-teams-toolkit-for-visual-studio-code)
+
+<!-- /code_chunk_output -->
+
+## Summary
+
+This sample shows how to integrate your Azure AI Search index as a data source into prompt templates through the Azure Chat Completions API.
+
+### Example Interaction
+
+![example interaction](assets/example.png)
+
+## Setting up the sample
+
+> [!NOTE]
+> Please note that at this time, this sample is only supported with Azure OpenAI.
+
+1. Clone the repository
+
+    ```bash
+    git clone https://github.com/Microsoft/teams-ai.git
+    ```
+
+2. Duplicate the `sample.env` in the `teams-ai/python/samples/04.ai.f.dataSource.azureOpenAI` folder. Rename the file to `.env`. 
+
+3. Please note that at this time, this sample is only supported with Azure OpenAI. Fill the `AZURE_OPENAI_KEY`, `AZURE_OPENAI_ENDPOINT` variables appropriately.
```

Comment:
To avoid confusion for devs, could you remove any extra logic in this sample that associates with OPENAI_KEY?

Reply:
removed in bot.py